site/docs/pages/swap/build-swap-transaction.mdx

```diff
@@ -0,0 +1,117 @@
+# `buildSwapTransaction`
+
+The `buildSwapTransaction` function is used to get an unsigned transaction for a swap between two Tokens.
+
```

Comment:
@0xAlec let's add more info here, like how to use `buildSwapTransaction` with `sendTransaction` from Wagmi. Also we should link `transaction` to `Transaction` the type.

So in our docs when we talk about entity we link the Types, so it makes sense on what are those.

Same for `Token`.